dhall-json/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+1.6.3
+
+* [yaml: Single-quote date/bool string fields](https://github.com/dhall-lang/dhall-haskell/commits/master/dhall-json)
+    * If you build against `aeson-yaml` (the default) these fields will now
```

Comment:
This sounds a bit as if `aeson-yaml` was an optional dependency for `dhall-json`.

Reply:
Oops!  I keep forgetting that we split out `dhall-yaml` into a separate package